Sources/DistributedSystem/DistributedSystem.swift

```diff
@@ -20,6 +20,10 @@ import struct Foundation.UUID
 internal import NIOCore
 internal import NIOPosix
 
+#if os(Linux)
```

Comment:
I would prefer do not have unnecessary imported dependencies if they are not really needed.
In that particular case we could use `!os(MacOS)` as well, but as soon as we have the third case (Windows) it would be wrong without testing which I have no time for now.